scripts/nithya_yetong00_constrainedopt_benchmark.cpp

```diff
@@ -0,0 +1,96 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  nithya_yetong00_constrainedopt_benchmark.cpp
```

Comment:
Just nithya :-) And, `constrained_opt_benchmark`.

Reply:
Done